tests/IntegrityTests/ValidatePrereleaseTxt.cs

```diff
@@ -74,7 +74,7 @@ public void VerifyProjectUsingPrereleaseComponentMarkedWithPrereleaseTxt()
             while (dirPath.Length >= TestSetup.DocsRootPath.Length)
             {
                 string dirName = Path.GetFileName(dirPath);
-                if (Regex.IsMatch(dirName, @"_(All|\d(\.\d)?)$"))
+                if (Regex.IsMatch(dirName, @"_(All|\d+(\.\d+)?)$"))
```

Comment:
not really understanding too much about this code tbh but I guess this does skip patch versions intentionally?

Reply:
We don't (and shouldn't) create versioned directories for patch releases.